src/v0/destinations/amazon_audience/utils.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const sha256 = require('sha256');
+const lodash = require('lodash');
+const { ConfigurationError, OAuthSecretError } = require('@rudderstack/integrations-lib');
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  getSuccessRespEvents,
+  removeUndefinedAndNullAndEmptyValues,
+} = require('../../util');
+
+// Docs: https://developer.x.com/en/docs/x-ads-api/audiences/api-reference/custom-audience-user
```

Comment:
This is twitter docs can you please update the reference?

Reply:
done